opensrp-chw/src/kk/assets/json.form/family_details_remove_member.json

```diff
@@ -101,16 +101,22 @@
         "values": [
           "{{family_details_remove_member.step1.remove_reason.values[0]}}",
           "{{family_details_remove_member.step1.remove_reason.values[1]}}",
-          "{{family_details_remove_member.step1.remove_reason.values[2]}}"
+          "{{family_details_remove_member.step1.remove_reason.values[2]}}",
+          "{{family_details_remove_member.step1.remove_reason.values[3]}}",
+          "{{family_details_remove_member.step1.remove_reason.values[4]}}"
         ],
         "keys": [
           "Died",
           "Moved away",
+          "Duplicate",
+          "Mistake entry",
           "Other"
         ],
         "openmrs_choice_ids": {
```

Comment:
Why are these ids in that weird language? How will they map to the keys?

Reply:
Yes, for the previous implementation those are the exact openmrs IDs for the choices, the ones that do not have the IDs are the ones that we have added so presumably if the data is to be pushed to openmrs the previous options will be the same and only 2 new options that have been introduces will have the normal readable ids that we introduced but as far as the data will be in opensrp server it will use the "keys" instead of the openmrs ids